src/rocks_util.h

```diff
@@ -48,6 +48,10 @@ namespace mongo {
         return nextPrefix;
     }
 
+    std::string encodePrefix(uint32_t prefix);
+    bool extractPrefix(const rocksdb::Slice& slice, uint32_t* prefix);
+    int get_internal_delete_skipped_count();
```

Comment:
These are small enough that we can make them `inline`, but no need to do that now, only if you feel like it in the next PR.

Reply:
I'll do it in the next PR, np.